config/jobs/image-pushing/k8s-staging-cluster-api.yaml

```diff
@@ -30,3 +30,61 @@ postsubmits:
           - name: creds
             secret:
               secretName: deployer-service-account
+    - name: post-cluster-api-provider-aws-push-images
+      cluster: test-infra-trusted
+      annotations:
+        # this is the name of some testgrid dashboard to report to.
+        testgrid-dashboards: sig-cluster-lifecycle-image-pushes
+      decorate: true
+      branches:
+        - ^master$
+      spec:
+        containers:
+          - image: gcr.io/k8s-testimages/image-builder:v20190906-d5d7ce3
+            command:
+              - /run.sh
+            args:
+              # this is the project GCB will run in, which is the same as the GCR images are pushed to.
+              - --project=k8s-staging-cluster-api
+              - --scratch-bucket=gs://k8s-staging-cluster-api-aws-gcb
```

Comment:
This should just be `<project>-gcb`. It's fine for multiple jobs to use the same bucket, if they're going into the same GCR.

Reply:
Definitely a mistake, good catch!